ledger/acctupdates.go

```diff
@@ -527,9 +527,11 @@ func (au *accountUpdates) onlineTop(rnd basics.Round, voteRnd basics.Round, n ui
 	}
 }
 
-// GetCreatorForRound returns the creator for a given asset/app index at a given round
-func (au *accountUpdates) GetCreatorForRound(rnd basics.Round, cidx basics.CreatableIndex, ctype basics.CreatableType) (creator basics.Address, ok bool, err error) {
-	return au.getCreatorForRound(rnd, cidx, ctype, true /* take the lock */)
+// GetCreator returns the creator for a given asset/app index at a given round
+func (au *accountUpdates) GetCreator(cidx basics.CreatableIndex, ctype basics.CreatableType) (creator basics.Address, ok bool, rnd basics.Round, err error) {
+	creator, ok, err = au.getCreatorForRound(au.latest(), cidx, ctype, true /* take the lock */)
+	rnd = au.latest()
```

Comment:
latest could have advanced between these two calls — could you move rnd = au.latest() up like
```suggestion
	rnd = au.latest()
	creator, ok, err = au.getCreatorForRound(rnd, cidx, ctype, true /* take the lock */)
```